client/components/use-case-wrapper.js

```diff
@@ -35,85 +33,96 @@ export function UseCaseWrapper({
-            </Stack>
-            {description && <div className="UsecaseWrapper_helper">{description}</div>}
-            {!hideDivider && <Divider className="UsecaseWrapper_divider" />}
-            <ul className="UsecaseWrapper_notes">
-              {listItems?.map((item, index) => (
-                <li key={index}>{item}</li>
-              ))}
-              {showAdminLink && (
-                <li>
-                  You can reset this scenario on the <Link href="/admin">Admin page</Link>.
-                </li>
-              )}
-              {articleURL && (
-                <li>
-                  Learn more about this scenario in the{' '}
-                  <a href={articleURL} target="_blank" rel="noreferrer">
-                    {title}
-                  </a>{' '}
-                  article.
-                </li>
-              )}
-              {!hideSrcListItem && (
-                <li>
-                  See the source code for this and other use cases{' '}
-                  <a
-                    href="https://github.com/fingerprintjs/fingerprintjs-pro-use-cases"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    on Github <GitHubIcon fontSize="small" />
-                  </a>
-                  .
-                </li>
-              )}
-            </ul>
-          </div>
-          <Paper
-            className="UsecaseWrapper_content"
-            variant="elevation"
+    <Paper variant="outlined" square sx={{ minHeight: '95vh', pb: (t) => t.spacing(2)}}>
+      <Box
+        sx={{
+          padding: (theme) => theme.spacing(4),
+          maxWidth: '600px',
+          margin: '0 auto',
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+        }}
+      >
+        <Stack direction="row" alignItems="baseline" spacing={1} justifyContent={'center'}>
+          {returnUrl && (
+            <Link href={returnUrl} legacyBehavior passHref>
+              <Tooltip title="Go back">
+                <IconButton component="a">
+                  <ArrowBack />
+                </IconButton>
+              </Tooltip>
+            </Link>
+          )}
+          <Typography
+            variant="h1"
             sx={{
-              padding: (theme) => theme.spacing(4),
+              textAlign: 'center',
+              pt: (t) => t.spacing(2),
             }}
           >
-            {children}
-          </Paper>
-        </div>
-      </div>
+            {title}
+          </Typography>
+        </Stack>
+        {description && (
+          <Box
+            sx={{
+              marginBottom: (t) => t.spacing(2),
+              // @ts-ignore
+              color: (t) => t.palette.gray,
+            }}
+          >
+            {description}
+          </Box>
+        )}
+        {!hideDivider && <Divider sx={{ width: '100%' }} />}
+        <ul style={{ lineHeight: '1.5rem', fontSize: '0.85rem' }}>
```

Comment:
Question - can we replace this with `List` component from mui? It will probably make it look more consistent with the rest of the UI

Reply:
Done, although `List` by default is not really meant for basic bullet point lists, I had to apply custom styles to make it look similar to before. What specifically did you imagine by "more consistent"?

Reply:
Overall I was hoping that by using it we can reduce amount of custom styles that we have to do. But in that case, if it's too much work let's leave it as it is

Reply:
This is what `List` looks like without custom styles: 
![image](https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/assets/11249993/31685da7-14f9-4c35-8b47-4c6a0ba9ae1c)

But the custom styles I applied are pretty minimal and using the <List> does allow us to use `theme` in the styling, so you could argue the change still makes sense for consistency sake :)